forms-endpoints.md

```diff
@@ -92,6 +90,30 @@ __Supported Query Types:__
  Currently default is starts_with (for most cases)
 
 __Supported Data Type__
+
+The value of `type` is one of the following:
+* `BOOLEAN`
+* `DATE`
+* `DOUBLE`
+* `INTEGER`
+* `MONEY`
+* `TEXT`
+* `VARCHAR`
```

Comment:
What file did you dig this list out of? 

I'm having some trouble verifying if this list is complete.

Reply:
Search for `BOOLEAN` and you'll find it.

Reply:
Path is in the proofgov repo at `lib/jsonb_query_builder.rb`